lib/views/reviews-view.js

```diff
@@ -644,4 +648,23 @@ export default class ReviewsView extends React.Component {
 
     return {lineNumber, positionText};
   }
+
+  /* istanbul ignore next */
+  scrollToThread(threadID) {
+    const threadHolder = this.threadHolders.get(threadID);
+    if (threadHolder) {
+      threadHolder.map(element => {
+        element.scrollIntoViewIfNeeded();
+        return null; // shh, eslint
```

Comment:
I really need to make a `RefHolder::forEach` method to make these cases nicer...